app/src/main/res/values/dimens.xml

```diff
@@ -596,6 +596,12 @@
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">160dp</dimen>
 
+<!--  RevisionCardFragment-->
```

Comment:
Spacing is incorrect
The comment should start from same position as that of below mentioned values and also there should be once space before and after the text
`<!-- RevisionCardFragment -->`

Reply:
The spacing and position corrected